test/regression_tests.py

```diff
@@ -11,26 +12,77 @@
 from ts_scripts.api_utils import test_api
 from ts_scripts.install_from_src import install_from_src
 from ts_scripts.regression_utils import test_regression
-from ts_scripts.utils import check_python_version
+from ts_scripts.utils import check_python_version, try_and_handle
 
-now = datetime.datetime.now()
-print("Current date and time : " + now.strftime("%Y-%m-%d %H:%M:%S"))
 
-check_python_version()
+def regression_tests(binaries, pypi, conda, nightly):
+    now = datetime.datetime.now()
+    print("Current date and time : " + now.strftime("%Y-%m-%d %H:%M:%S"))
 
-# Install from source
-install_from_src()
+    check_python_version()
 
-# Generate mar file
-mg.generate_mars()
+    if binaries:
+        if pypi:
+            if nightly:
+                cmd = f"pip install torchserve-nightly torch-model-archiver-nightly torch-workflow-archiver-nightly"
+            else:
+                cmd = f"pip install torchserve torch-model-archiver torch-workflow-archiver"
+        elif conda:
+            if nightly:
+                cmd = f"conda install -c pytorch-nightly torchserve torch-model-archiver torch-workflow-archiver"
+            else:
+                cmd = f"conda install -c pytorch torchserve torch-model-archiver torch-workflow-archiver"
+        print(f"## In directory: {os.getcwd()}; Executing command: {cmd}")
+        try_and_handle(cmd, False)
+    else:
+        # Install from source
+        install_from_src()
 
-# Run newman api tests
```

Comment:
could you plz elaborate on the needed changes for the newman tests?

Reply:
I am not sure i follow. There is no change to the tests. I am only changing how we install torchserve

Reply:
Git diff makes it seem like I have changed the tests